### ISSUE TITLE
Tie orders directly to element_id

### ISSUE THREAD

**Issue by jessevondoom** [cashmusic/platform#288]
Current behavior:
We associate orders with a specific session, but do not directly associate them with an element id. This doesn't affect fulfillment or behavior, but makes it hard to measure the impact of one element over another. 

Desired behavior: 
We add an optional element_id column to the order, and pass the element id to set when creating an order.

**Comment by christopherleckie** [comment]
Should also bring through a referral URL along with the element ID for orders so we are aware of where the user ordered from.

**Comment by jessevondoom** [comment]
Good call. We already grab the URL where the element was embedded so if we included that along with the ID there's some serious intelligence we can capture.
